net/aardvark-dns/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=aardvark-dns
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/containers/aardvark-dns.git
+PKG_SOURCE_DATA:=2025-05-12
```

Comment:
```suggestion
PKG_SOURCE_DATE:=2025-05-12
```

Reply:
Fixed as proposed, except changed year from 2025 -> 2023 while at it :)